grails-app/conf/Config.groovy

```diff
@@ -167,6 +167,7 @@ log4j.main = {
 	warn 'grails.app'
 //    debug 'grails.app'
 //    debug 'liquibase'
+//    debug 'org.bbop.apollo'
```

Comment:
can probably remove, unless you want to leave it for reference.  I think you want:

```debug 'grails.app````